README.md

```diff
@@ -394,6 +394,79 @@ This creates a Python entrypoint, YAML configs and a Dockerfile. You can start f
 
 </details>
 
+#### MultimodalDocument
+  
+ A MultimodalDocument in Jina is a document composed by more than one chunks with different modalities.
```

Comment:
I wonder if we should use "input" instead of "document" for people not familiar with Jina terminologies.

Reply:
@yuanbit i think at this point we've used the word Document so much that people should have a feel for it, especially if they're reading advanced stuff like multi-modal search

Other notes:
- `document` should be `Document` (bc it refers to a Jina-specific concept, not a document in general)
- `chunk` should be `Chunk` (same reason)
- `more than one` -> `multiple` (easier to read, and relates to the "multi" part)

Also, first time we mention each, let's link to the relevant section of Jina 101 that describes them. That gets around folks not understanding our usage of Document